resources/models/reservation.py

```diff
@@ -119,9 +119,9 @@ def is_active(self):
     def need_manual_confirmation(self):
         return self.resource.need_manual_confirmation
 
-    def are_extra_fields_visible(self, user=None):
+    def are_extra_fields_visible(self, user):
         if not self.need_manual_confirmation():
```

Comment:
Yeah, because we want to limit extra field visibility based on the user only on reservations that need manual confirmation.